Sources/PackageDescription4/Package.swift

```diff
@@ -134,24 +146,24 @@ public final class Package {
             }
         }
 
-        /// The url of the dependency.
+        /// The Git url of the package dependency.
         public let url: String
 
-        /// The dependency requirement.
+        /// The dependency requirement of the package dependency.
         public let requirement: Requirement
 
-        /// Create a dependency.
+        /// Initializes and returns a newly allocated requirement with the specified url and requirements.
         init(url: String, requirement: Requirement) {
             self.url = url
             self.requirement = requirement
         }
     }
 
-    /// The name of the package.
+    /// The name of the Swift package.
     public var name: String
 
   #if !PACKAGE_DESCRIPTION_4
-    /// The list of platforms supported by this package.
+    /// The list of platforms that this Swift package supports.
```

Comment:
This doc comment is outdated. Maybe "This list of supported platforms with customized deployment target"